test/binaries/test-openssh-7.7p1.c

```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+
+int main() {
+	printf("This program is designed to test the cve-bin-tool checker.");
+	printf("It outputs a few strings normally associated with OpenSSH 7.7p1");
+	printf("They appear below this line.");
+	printf("------------------");
+	printf("OpenSSH_7.7p1");
```

Comment:
As with the other checkers, please change the tests so they are not looking for the same version so that we don't hit the test_scanner bug.